its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/OrchestratorState.java

```diff
@@ -48,16 +49,10 @@ public void startOnce() throws Exception {
       usageCount += 1;
       if (usageCount == 1) {
         orchestrator.start();
-        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfile.xml"));
-        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileCpp.xml"));
-        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileExternalIssuesVB.xml"));
-        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileExternalIssues.xml"));
-        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileCSharp.xml"));
-        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileVBNet.xml"));
-        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileParameters.xml"));
-        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/ExcludedTestQualityProfile.xml"));
-        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestEmptyQualityProfile.xml"));
-
+        for (String profile: QualityProfiles.allProfiles())
```

Comment:
You can `var` now :)
```suggestion
        for (var profile: QualityProfiles.allProfiles())
```